apps/documentation/src/components/MdxProvider/MdxTitle/MdxTitle.tsx

```diff
@@ -14,28 +14,22 @@ export default function MdxTitle(props: TitleProps): JSX.Element {
 }
 
 function idGen(children: ReactNode): string {
```

Comment:
How about naming this function `getIdFrom` so when you can read its source you get something  
"get id from children"?

```suggestion
function getIdFrom(children: ReactNode): string {
```